src/styles.css

```
@@ -534,50 +541,30 @@
+   surface past Typeset's own @layer components. --typeset-font-heading has to be
+   named even though it wants the body face: it defaults to var(--font-heading),
+   which this app does not define, and an undefined custom property inside
+   font-family makes the declaration invalid rather than inherited. */
+.typeset-note {
+  --typeset-font-body: var(--font-editor);
+  --typeset-font-heading: var(--font-editor);
+  --typeset-font-mono: var(--font-mono);
+  --typeset-size: 1rem;
+  --typeset-leading: 1.75;
+  --typeset-flow: 1.25em;
   font-optical-sizing: auto;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)